app/Community/Actions/UpdateGameClaimAction.php

```diff
@@ -0,0 +1,131 @@
+
+        if (array_key_exists('type', $newValues)) {
+            $newType = (int) $newValues['type'];
+            if ($claim->ClaimType !== $newType) {
+                $claim->ClaimType = $newType;
+
+                $auditMessage .= " Claim Type: " . ClaimType::toString($newType) . '.';
+            }
+        }
+
+        if (array_key_exists('set_type', $newValues)) {
+            $newType = (int) $newValues['set_type'];
+            if ($claim->SetType !== $newType) {
+                $claim->SetType = $newType;
+
+                $auditMessage .= " Set Type: " . ClaimSetType::toString($newType) . '.';
+            }
+        }
+
+        if (array_key_exists('status', $newValues)) {
+            $newStatus = (int) $newValues['status'];
+            if ($claim->Status !== $newStatus) {
+                $claim->Status = $newStatus;
+
+                $auditMessage .= " Claim Status: " . ClaimStatus::toString($newStatus) . '.';
+
+                if (!ClaimStatus::isActive($newStatus)) {
+                    $claim->Finished = Carbon::now();
+
+                    if ($newStatus === ClaimStatus::Complete) {
+                        $this->processCompletedClaim($claim, $currentUser);
+                    }
+                }
+            }
+        }
+
+        if (array_key_exists('special', $newValues)) {
+            $newSpecial = (int) $newValues['special'];
+            if ($claim->Special !== $newSpecial) {
+                $claim->Special = $newSpecial;
+
+                $auditMessage .= " Special: " . ClaimSpecial::toString($newSpecial) . '.';
+            }
+        }
+
+        if (array_key_exists('claimed', $newValues)) {
+            $newDate = $newValues['claimed'];
+            if ($claim->Created->notEqualTo($newDate)) {
+                $claim->Created = $newDate;
+
+                $auditMessage .= " Claim Date: $newDate.";
+            }
+        }
+
+        if (array_key_exists('finished', $newValues)) {
+            $newDate = $newValues['finished'];
+            if ($claim->Finished->notEqualTo($newDate)) {
+                $claim->Finished = $newDate;
+
+                $auditMessage .= " End Date: $newDate.";
+            }
+        }
+
+        if ($claim->isDirty()) {
+            $claim->save();
+
+            addArticleComment("Server", ArticleType::SetClaim, $claim->game_id, $auditMessage);
+        }
+    }
+
+    private function processCompletedClaim(AchievementSetClaim $claim, User $currentUser): void
+    {
+        addArticleComment("Server", ArticleType::SetClaim, $claim->game_id, "Claim completed by {$currentUser->display_name}");
+
+        // also complete any collaboration claims
+        $game = Game::find($claim->game_id);
+        $game->achievementSetClaims()->active()->update(['Status' => $claim->Status, 'Finished' => $claim->Finished]);
+
+        // TODO: these emails should be queued and sent asynchronously
```

Comment:
I've tried various ways to optimize this locally, and I think I've landed at the same place as this comment.

It seems like we either need a bulk email send method, which I'm not even sure how to configure and/or implement, or we probably need a dedicated queue for these specific emails.

Either way, it's not a behavior change from what's happening in `master`.